webapp/src/modules/translation/locales/es.json

```
@@ -635,25 +637,43 @@
       "notice": "Estamos en mantenimiento, volveremos pronto!"
     }
   },
-  "create_rental_modal": {
-    "title": "Poner en alquiler",
-    "price": "Precio por día",
-    "expiration_date": "Fecha de expiración del clasificado",
-    "periods": "Período del alquiler",
-    "periods_tooltip": "Selecciona la cantidad de días por las que estas dispuesto a alquilar tu tierra",
-    "notice_line_one": "Antes de publicar tu tierra en alquiler debes aprobar al contrato de {link} para que pueda operar con tus tierras, exactamente igual que cuando intentas venderla.",
-    "notice_line_two": "Cuando el periodo de alquiler se termina, tienes la opción de extenderlo o de reclamar tu tierra de vuelta, si decides reclamarla, deberás enviar una nueva transacción para esto.",
-    "notice_link": "Alquiler",
-    "period_options": {
-      "one_week": "7 días",
-      "one_month": "30 días",
-      "two_months": "60 días",
-      "quarter_year": "90 días",
-      "half_year": "180 días",
-      "one_year": "365 días"
+  "rental_modal": {
+    "authorization_step": {
+      "title": "Poner en alquiler",
+      "notice_line_one": "Antes de listar tu {assetType} en alquiler necesitas autorizar al contrato de {link} a operar tu {assetType}, de la misma forma que cuando deseas vender tu {assetType}.",
+      "notice_line_two": "Cuando el período de alquiler termine tiene dos opciones :",
```

Does it requires a whitespace or is it a typo?
```suggestion
      "notice_line_two": "Cuando el período de alquiler termine tiene dos opciones:",
```